packages/astro/src/core/build/plugins/plugin-chunks.ts

```diff
@@ -0,0 +1,31 @@
+import type { Plugin as VitePlugin } from 'vite';
+import type { AstroBuildPlugin } from '../plugin.js';
+import { extendManualChunks } from './util.js';
+
+export function vitePluginChunks(): VitePlugin {
```

Comment:
can you add some comments about what these 2 exports functions are file? It doesn't seem quite clear to me.